### ISSUE TITLE
Powerqueen MPPT LiFePo4 battery charger

### ISSUE THREAD

**Issue by Keyronics** [ioBroker/AdapterRequests#835]
**What kind of device or service would you like to see an adapter for?**
Powerqueen MPPT Laderegler
https://www.ipowerqueen.de/products/power-queen-30a-mppt-solarladeregler-solar-laderegler-mit-einstellbarer-lcd-anzeige

**Is the device connected to the internet or only available on a local network?**
Local via Bluetooth

**Is an official App or Website available?**
https://apps.apple.com/de/app/solar-app/id6444056453

**Is an official API including documentation available?**
No

**Are other libraries for an integration available?**
I´m not sure

**Is this device already integrated in other Smart Home systems?**
I´m not sure

**Is this device already integrated in homebridge? Might the ham adapter in combination with the homebridge plugin be sufficient?**
No

**Additional context**
The connection is important to see the battery voltage and the power of the suncollector.

**Comment by Keyronics** [comment]
Yes, thats right. The Bluetooth Link Module is connected to the rs232 socket.
Bluetooth would be great. I´m living in a rented flat and try to avoid drilling holes from the inside to outside.
Can I help somehow?

**Comment by tschamad** [comment]
the new version of the controller has bluetooth direct on board.
